### PR TITLE
documentation added to angular

### PR DESCRIPTION
**Pull request by radioheadftw**
There are a few areas where the documentation could be improved, but I need to get what I have pushed to the cloud-dev branch so others working on documentation can provide their input.

### CODE REVIEW

File: documentation/coverage.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        <div class="container-fluid main">
+           <div class="row main">
+               <div class="hidden-xs menu">
+                   <compodoc-menu mode="normal"></compodoc-menu>
+               </div>
+               <!-- START CONTENT -->
+               <div class="content coverage">
+                   <div class="content-data">
+
+
```
Comment (HugoMnFernandez) [inline documentation/coverage.html]: A little bit to much blank spaces

File: documentation/coverage.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    <div class="has-results">
+        <h1 class="search-results-title"><span class='search-results-count'></span> result-matching "<span class='search-query'></span>"</h1>
+        <ul class="search-results-list"></ul>
+    </div>
+    <div class="no-results">
+        <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
+    </div>
+</div>
+</div>
+               <!-- END CONTENT -->
```
Comment (HugoMnFernandez) [inline documentation/coverage.html]: Weird indentation

Comment (HugoMnFernandez) [reply documentation/coverage.html]: Should be fixed with a good ol' alt+shift+f

File: documentation/dependencies.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    </div>
+    <div class="no-results">
+        <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
+    </div>
+</div>
+</div>
+               <!-- END CONTENT -->
+           </div>
+       </div>
+
```
Comment (HugoMnFernandez) [inline documentation/dependencies.html]: Weird indentation with the divs, try automatic formating

File: documentation/index.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+       <script src="./js/compodoc.js"></script>
+
+       <script src="./js/tabs.js"></script>
+       <script src="./js/menu.js"></script>
+       <script src="./js/libs/clipboard.min.js"></script>
+       <script src="./js/libs/prism.js"></script>
+       <script src="./js/sourceCode.js"></script>
+          <script src="./js/search/search.js"></script>
+          <script src="./js/search/lunr.min.js"></script>
```
Comment (HugoMnFernandez) [inline documentation/index.html]: The same, watch out indentation, alt+shift+f

File: documentation/index.html [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    </div>
+    <div class="no-results">
+        <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
+    </div>
+</div>
+</div>
+               <!-- END CONTENT -->
+           </div>
+       </div>
+
```
Comment (HugoMnFernandez) [inline documentation/index.html]: Again this weird indentation, is it auto-generated content?

File: src/app/services/reservation/reservation.service.ts [hunk 6]
```diff
@@ -12,53 +12,75 @@ import { UserService } from '../user/user.service';
...
 
-  ///////////////////////////////////////////////////
-  // Methods pertaining to objects that need to be
-  // shared among various components
-  //////////////////////////////////////////////////
+  /**
+   * Methods pertaining to objects that need to be
+   * shared among various components
+   */
+
```
[the remark this answers is not in the document]
Comment (nuklearpineapple) [reply src/app/services/reservation/reservation.service.ts]: Potentially use region instead

**Pull request merged**